### PR TITLE
Add kirbytext hook order recipe and links

### PR DESCRIPTION
**Pull request by texnixe**

### CODE REVIEW

File: content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Title: The order of kirbytext/kirbytags hooks
```
Comment (lukasbestle) [inline content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt]: The terms are used in mixed casing throughout the PR. According to our content styleguide, it should be `KirbyText` and `KirbyTags`.

File: content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+The `kirbytext:before` hook is triggered before any parsing is applied to the raw text. At this state, you can therefore manipulate the raw text of the field.
+
+## `kirbytags:before`
+
+The `kirbytags:before` hook kicks in directly afterwards, i.e. before KirbyTags are parsed, but after modifications through a `kirbytext:before` hook.
+
+## `kirbytags:after`
+
+The `kirbyTags:after` hook kicks in after modifications through a `kirbytags:before` hook and after all KirbyTags are parsed, but before Markdown and Smartypants are parsed.
```
Comment (lukasbestle) [inline content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt]: Smartypants -> SmartyPants

Also in the following paragraph.

**Review by lukasbestle — COMMENTED** [review]
I really like how it's a short and simple recipe that gives a lot of helpful background information. I'd only suggest a few minor changes to wording.

File: content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt (hunk 1 above)
Comment (texnixe) [reply content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt]: Hm, the hooks itself are called `kirbytext:before` etc. in the docs, without mixed case, so I wasn't sure what to use where.

**Comment by texnixe** [comment]
Thanks for reviewing, @lukasbestle

File: content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt (hunk 1 above)
Comment (lukasbestle) [reply content/1_docs/2_cookbook/7_extensions/0_kirbytext-hooks/recipe.txt]: The hooks yes, but otherwise I‘d use the uppercase forms.

**Review by bastianallgeier — APPROVED** [review]
Awesome!! Short and super useful!

**Pull request merged**